frontend/src/views/Game.tsx

```diff
@@ -1,13 +1,44 @@
 import Editor from '@monaco-editor/react';
 import React from 'react';
+import { Problem, getProblems } from '../api/problem';
+import styles from './styles.module.css';
 
-function GamePage() {
-  return (
-    <div>
-      Game page
-      <Editor height="90vh" language="javascript" />
-    </div>
-  );
+interface GameState {
+  problems?: Problem[];
+}
+
+interface GameProps {}
+
+class GamePage extends React.Component<GameProps, GameState> {
+  constructor(props: GameProps) {
+    super(props);
+    this.state = {};
+  }
+
+  componentDidMount() {
+    getProblems().then((problems) => {
+      this.setState({ problems });
+    }).catch((error) => {
+      console.log(error);
+    });
+  }
+
+  render() {
+    const { problems } = this.state;
+    const firstProblem = problems?.[0];
+    return (
+      <div>
+        <div className={styles.header}><h3>Game page</h3></div>
+        <div className={styles.problemPanel}>
+          <h3>{ firstProblem?.name }</h3>
+          <p>{ firstProblem?.description}</p>
+        </div>
+        <div className={styles.editorPanel}>
+          <Editor height="90vh" language="javascript" />
```

Comment:
Note: A scrollbar appears next to the `Editor` when you add new lines, even when it doesn't go past the bottom of the page. Is this fixable?

Reply:
It looks like there's no way to make it disappear completely even while turning its visibility to "hidden". There doesn't seem to be an option to remove the large chunk of white space under the last line either.

Reply:
Maybe that's part of the desired behavior? It seems like it makes it so that you can always scroll down just enough so that you don't see any of the lines you typed (IIRC VSCode does the same). Idk if there's a way to toggle that or not.

Reply:
I'll make a note to look more into this issue when we're working more on the design! I think what's happening is there's an empty block under the code editor that stays the same size even when the editor expands.